src/router/index.js

```diff
@@ -139,11 +118,39 @@ const routes = [
     meta: { title: "Organizations Page", requiresAuth: true },
     component: () => import("../views/Organizations/Organizations.vue"),
   },
+  {
+    path: "/organization/:organizationId",
```

Comment:
As mentioned in the previous review, let's make this organization**s** to keep our routes consistent.
```suggestion
    path: "/organizations/:organizationId",
```

Reply:
done